### PR TITLE
Adding Niu Mini to VIA

### PR DESCRIPTION
**Pull request by xplusplus**
## Description

Porting the Niu Mini keyboard to VIA

## Types of Changes

- [x] Adding a keyboard to VIA
- [ ] Changing a keyboard already in VIA
- [ ] Bug fix
- [ ] Documentation

## QMK Pull Request

Merged: https://github.com/qmk/qmk_firmware/pull/8702

## Checklist

- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] I have followed the style guide here: https://caniusevia.com/docs/layouts

### CODE REVIEW

File: src/other/niu_mini/niu_mini.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{
+  "name":"niu_mini",
+  "vendorId":"0xFEED",
+  "productId":"0x6060",
```
Comment (Wilba6582) [inline src/other/niu_mini/niu_mini.json:4]: Please change the vendor and product IDs

**Comment by Wilba6582** [comment]
Please fix the accent keys, they are black background

File: src/other/niu_mini/niu_mini.json (hunk 1 above)
Comment (xplusplus) [reply src/other/niu_mini/niu_mini.json:4]: Is there general guidance on allocating vendor and product IDs for VIA/QMK, or do I just find values that haven't been used and checked in yet?

Comment (Wilba6582) [reply src/other/niu_mini/niu_mini.json:4]: See here: https://caniusevia.com/docs/configuring_qmk#change-vendor_id-product_id

I suggest pick a unique one for the vendor, and start numbering product at 0x0001

Comment (xplusplus) [reply src/other/niu_mini/niu_mini.json]: updated

**Comment by xplusplus** [comment]
Fixed accent colors.

**Comment by xplusplus** [comment]
@Wilba6582 qmk/qmk_firmware#8702 has beeb merged

**Pull request merged**